service/worker/diagnostics/invariants/types.go

```diff
@@ -37,13 +37,28 @@ const (
 	TimeoutTypeChildWorkflow TimeoutType = "Child Workflow Execution has timed out"
 )
 
+type RootCause string
```

Comment:
I wonder if there is a way to avoid this type.
Big portion of the diff is `.String` conversion (30+ times).

Reply:
we can use alias in this case
```suggestion
type RootCause = string
```

Reply:
updated to follow the suggestion